apim-certifications/backends/books-service/.gitignore

```diff
@@ -0,0 +1,31 @@
+HELP.md
```

Comment:
Any reasons to ignore HELP.md ?

Reply:
Removed via fbe6a1d0f09571a6eaee2f5e47704b7df7b09f46